common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -37,7 +37,7 @@ define('video/09_events_plugin.js', [], function() {
         initialize: function() {
             this.events = {
                 'ready': this.onReady,
-                'play': this.onPlay,
+                'play_video': this.onPlay,
```

Comment:
I think a better name for this event will be `play_started` since it is triggered as a notification. The others are instructions.